src/controller/index.ts

```diff
@@ -63,7 +66,9 @@ export default class SimulariumController {
         };
 
         /* eslint-disable */
-        this.handleTrajectoryInfo = (msg: TrajectoryFileInfo) => {
+        this.handleTrajectoryInfo = (
```

Comment:
just curious, is the eslint-disable still needed here?

Reply:
Yeah, I just tried removing it and we get a few warnings about unused arguments.